docs/gh_pages/docs/retrieval_chain.md

```diff
@@ -27,7 +27,7 @@ The walkthrough requires Langchain, langchain-community, langchain-openai, and a
 %pip install --upgrade --quiet  langchain langchain-community langchain-openai qdrant_client
 ```
 
-### Identity-aware Data Ingestion
+## Identity-aware Data Ingestion
```

Comment:
_Identity-Aware Data Ingestion_ should be an H3; it's part of the setup.

Reply:
Done